lib/utils/twacode.dart

```diff
@@ -700,7 +720,9 @@ class TwacodeRenderer {
   }
 
   List<InlineSpan> render(
-      {required List<dynamic> twacode, TextStyle? parentStyle}) {
+      {required List<dynamic> twacode,
+      TextStyle? parentStyle,
+      double? userUniqueColor}) {
```

Comment:
Same here, either make it required, or provide a default

Reply:
You're right, I will go through the entire file to check all the fields then for consistency